src/moustachu.nim

```diff
@@ -65,9 +66,9 @@ proc parallelReplace(str: string,
     result = result.replace(sub[0], sub[1])
 
 proc render(tmplate: string, contextStack: seq[Context], partialsDir="."): string =
-  ## Take a mustache template `tmplate`, an evaluation Context `c`, and an optional
-  ## path `partialsDir` (by default set to the current directory) which may contain partial files
-  ## and return the rendered string. This is the main procedure.
+  # Take a mustache template `tmplate`, an evaluation Context `c`, and an optional
+  # path `partialsDir` (by default set to the current directory) which may contain partial files
+  # and return the rendered string. This is the main procedure.
```

Comment:
We can keep the double ## (documentation comments), right?

Reply:
I'm under the impression that docgen comments are for public procedures. If you restore it to documentation comments, you might want to mention that it is private. Or, you could make it public. I don't see any harm in making it public.